src/hooks/custom-hooks/useTrackjs.ts

```diff
@@ -0,0 +1,22 @@
+import { TrackJS } from 'trackjs';
+import { useAuthData } from '@deriv-com/api-hooks';
+
+const { VITE_TRACKJS_TOKEN } = process.env;
+
+const useTrackjs = () => {
+    const { activeLoginid } = useAuthData();
+    const init = () => {
+        TrackJS.install({
+            application: 'deriv-p2p',
+            dedupe: false,
+            enabled: location.host.indexOf('localhost') !== 0,
+            token: VITE_TRACKJS_TOKEN!,
+            userId: activeLoginid || 'undefined',
```

Comment:
@farrah-deriv, could you please check if we can use the user's loginId as an identifier, or do we need to change it because of GDPR? as far as I checked it should be fine but need to double confirm it

Reply:
what will happen if we don't pass the userId?

Reply:
the default value is `undefined`

Reply:
based on our discussion , I think there is nothing to worry about. we are already using the loginid in trackjs for the users